car_time/time.go

```diff
@@ -57,12 +57,16 @@ func (c *Car) CanShow() bool {
 func (c *Car) Render(out chan<- string) {
 	defer close(out)
 	carPaint := ansi.ColorFunc(c.GetPaint())
-
 	n := time.Now()
+
+	t := n.Format("15:04:05")
+	if h := os.Getenv("BULLETTRAIN_CAR_TIME_12HR"); h == "true" {
+		t = n.Format("3:04:05")
+	}
+
 	out <- fmt.Sprintf("%s%s",
```

Comment:
This operation is now so small, could you shrink it back to a single line, please?

Reply:
Not sure what your looking for here?

Reply:
I wanted to say, the whole `out <- fmt.Sprintf ...` expression could fit into a single line now, since the parameter's length has shortened. Now it's taking up 3 lines.